src/python/pants/vcs/changed.py

```diff
@@ -104,11 +104,17 @@ class ChangedOptions:
 
     since: str | None
     diffspec: str | None
-    dependees: DependeesOption
+    dependents: DependentsOption
 
     @classmethod
     def from_options(cls, options: OptionValueContainer) -> ChangedOptions:
-        return cls(options.since, options.diffspec, options.dependees)
+        return cls(
+            options.since,
+            options.diffspec,
+            options.dependees
+            if options.dependents == DependentsOption.NONE
+            else options.dependents,
```

Comment:
FWIW: it doesn't seem strictly necessary in this case, but you can use `resolve_conflicting_options` for this: https://github.com/pantsbuild/pants/blob/521ee5fe06556233dcaaf1eea4cfe1d564f63642/src/python/pants/backend/python/subsystems/repos.py#L106-L113